nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -104,7 +104,7 @@ def admonition(self, text, kind):
         if kind not in admonitions:
             raise NotImplementedError(f"admonition {kind} not supported yet")
         tag = admonitions[kind]
-        return f"<{tag}><para>{text.rstrip()}</para></{tag}>"
+        return f"<para><{tag}>{text.rstrip()}</{tag}></para>"
```

Comment:
Is there a reason we need to wrap these elements in paragraphs? IIRC, they can be used stand-alone in most context `para` can.